crates/typst-library/src/math/frac.rs

```diff
@@ -4,26 +4,26 @@ use crate::diag::bail;
 use crate::foundations::{elem, Content, Value};
 use crate::math::Mathy;
 
-/// A mathematical fraction.
+/// 分数。
 ///
-/// # Example
+/// # 例
 /// ```example
 /// $ 1/2 < (x+1)/2 $
 /// $ ((x+1)) / 2 = frac(a, b) $
 /// ```
 ///
-/// # Syntax
-/// This function also has dedicated syntax: Use a slash to turn neighbouring
-/// expressions into a fraction. Multiple atoms can be grouped into a single
-/// expression using round grouping parenthesis. Such parentheses are removed
-/// from the output, but you can nest multiple to force them.
+/// # 構文
+/// この関数には専用の構文もあります。
+/// 隣接する式をスラッシュで区切ると、分数になります。
+/// また、丸括弧で複数の要素を囲うと、単一の式として扱えます。
+/// そのような丸括弧は出力からは削除されますが、複数の丸括弧をネストすることにより丸括弧を残すことができます。
```

Comment:
一文に「丸括弧」という単語が3つあり、「......ことにより......ことができます。」という文になっているため、少し修正してもよいかもしれません。
一例をあげます。

```suggestion
/// そのような丸括弧は出力からは削除されますが、複数重ねてネストすることで、丸括弧を表示させることも可能です。
```

Reply:
ありがとうございます！そのようにさせていただきます。